File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/SAT/consultar_sat.dart
import 'sat_command.dart';

class ConsultarSAT extends SatCommand {
  final int _numSessao;

  ConsultarSAT(this._numSessao) : super('ConsultarSat');

  @override
  functionParameters() {
    return '"numSessao"' ':' '$_numSessao';
  }
}

File: Exemplos M10/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/SAT/Commands/ConsultarSAT.ts
import { SatCommand } from "../SatCommand";

export class ConsultarSAT extends SatCommand {
  readonly numSessao: number;

  constructor(numSessao: number) {
    super("ConsultarSat");
    this.numSessao = numSessao;
  }

  functionParameters(): object {
    return { numSessao: this.numSessao };
  }
}
